### DISCUSSION TITLE
SSL Tools

### DISCUSSION THREAD

**Discussion by cipradu** [CorentinTh/it-tools#433] [Ideas]
Hi,

i think having some SSL toos in this collection would be useful ...
Something like 
- csr/key generator (2048/4096)
- csr/cert decoder
- pfx extractor
- pfx generator

I hope this makes it to the collection

Thank you for all the time and effort!

**Comment by sharevb** [comment]
Hi @cipradu, I made many PR related to this : #910, #1027 #887 #917 #888 #915

**Reply by douglasg14b** [reply]
PRs seem _very_ backed yup, what's the word on these?

**Reply by sharevb** [reply]
Hi, for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
